samples/gocdk-blob/blob.ct

```diff
@@ -18,19 +18,19 @@
 $ mkdir bucket
 
 # List an empty bucket: valid, but no output.
-$ gocdk-blob ls file://${ROOTDIR}/bucket
+$ gocdk-blob ls file://${SLASHDIR}/bucket
```

Comment:
This took me a bit to follow. Might it be more clear to use `ROOTDIR_URL` as the environment variable that is added in `Setup`, where the value is the full URL? Then this line, for example, would be:

```
$ gocdk-blob ls ${ROOTDIR_URL}/bucket
```

Up to you.

Reply:
done